src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/MigrationAssessmentsRequest.cs

```diff
@@ -16,20 +17,21 @@ public class MigrationAssessmentsParams
     public class MigrationAssessmentResult
     {
         /// <summary>
-        /// Gets the collection of assessment results.
+        /// Errors that happen while running the assessment
         /// </summary>
-        public List<MigrationAssessmentInfo> Items { get; } = new List<MigrationAssessmentInfo>();
-
+        public ErrorModel[] Errors { get; set; }
         /// <summary>
-        /// Gets or sets a value indicating
-        /// if assessment operation was successful.
+        /// Result of the assessment
         /// </summary>
-        public bool Success { get; set; }
-
+        public ServerProperties Result { get; set; }
+        /// <summary>
+        /// Start time of the assessment
+        /// </summary>
+        public string StartedOn { get; set; }
```

Comment:
nit: I would probably use names StartTime and EndTime, but that is mostly just preference